host/xtest/regression_4000.c

```diff
@@ -24,6 +24,9 @@
 
 #include <assert.h>
 
+#if defined(CFG_CRYPTOLIB_NAME_mbedtls)
```

Comment:
This variable is not exported by https://github.com/OP-TEE/optee_os/blob/3.13.0/ta/ta.mk#L28-L42 so how does this work?

Reply:
host/xtest do include _.../export-ta_arm[32|64]/host_include/conf.h_

Reply:
@etienne-lms ah, yes, everything is exported under host_include (as `.mk` and `.h`). Thanks for the reminder :)